lib/Common/Memory/HeapBucket.h

```diff
@@ -75,8 +75,12 @@ class HeapBucket
 protected:
     HeapInfo * heapInfo;
     uint sizeCat;
+    bool allocationsStartedDuringConcurrentSweep;
```

Comment:
>allocationsStartedDuringConcurrentSweep [](start = 9, length = 39)

why this is not in the following #if?

Reply:
It can be. I will do that change.

---
In reply to: [148181880](https://github.com/Microsoft/ChakraCore/pull/4104#discussion_r148181880) [](ancestors = 148181880)